src/main/kotlin/com/github/devcordde/devcordbot/listeners/DevmarktRequestUpdater.kt

```diff
@@ -134,7 +134,7 @@ class DevmarktRequestUpdater(
         builder
             .addField("Titel", "`$requestTitel`", true)
             .addField("Author", "`$requestAuthor`", true)
-            .addField("Begründung", "`" + reason.replace("deny:", "") + "`", false)
+            .addField("Begründung", "`" + reason.drop(6) + "`", false)
```

Comment:
you should ask for `startsWith("deny: ")` in ln 120 and do `.replace("deny: ", "")` here.